packages/electrode-react-webapp/lib/default-handlers.js

```diff
@@ -62,10 +62,23 @@ module.exports = function setup(options) {
       const data = context.$.data;
       const manifest = bundleManifest();
       const manifestLink = manifest ? `<link rel="manifest" href="${manifest}" />\n` : "";
-      const css = WEBPACK_DEV
-        ? data.devCSSBundle
-        : (data.cssChunk && `${prodBundleBase}${data.cssChunk.name}`) || "";
-      const cssLink = css && !data.criticalCSS ? `<link rel="stylesheet" href="${css}" />` : "";
+      const css = [];
+
+      if (WEBPACK_DEV) {
+        if (Array.isArray(data.devCSSBundle)) css.push(...data.devCSSBundle);
+        else if (data.devCSSBundle) css.push(data.devCSSBundle);
+      } else if (Array.isArray(data.cssChunk)) {
+        css.push(...data.cssChunk);
+      } else if (data.cssChunk) css.push(data.cssChunk);
```

Comment:
rather than `push`, `concat` would work better here and you don't have to do `isArray` check.